README.md

```diff
@@ -78,7 +78,7 @@ The client is built around the concept of sources; a set of packages effectively
 
 * Windows 10 1809 (17763) or later
 * [Developer Mode enabled](https://docs.microsoft.com/windows/uwp/get-started/enable-your-device-for-development)
-* [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/)
+* [Visual Studio](https://visualstudio.microsoft.com/downloads/)
```

Comment:
```suggestion
* [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/)
```
Because I think it won't build without 2022 now due to `std::hash` not being implemented for `std::filesystem::path` in 2019.

Reply:
I wasn't sure whether or not to specify the year, since the link will always lead to the latest version. When Visual Studio 2023 comes out, I was thinking this would save an edit later. But, since 2022 is the minimum required, it probably is best to call that out explicitly

Reply:
Chances are good that when VS Next comes out, it won't build without some changes either.  But yes, I wanted to call out that 2022 was in fact the minimum.